packages/example/src/pages/guides/styling.mdx

```diff
@@ -25,12 +25,12 @@ for you to share your component with other theme consumers down the line.
 Every Sass file in your project automatically has access to the the following
```

Comment:
Were you able to "auto-import" styles with Sass modules to mirror how the theme currently works?